.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: go
+
+go:
+- 1.9
+- tip
+
+before_install:
+- go get github.com/Masterminds/glide
+- go get github.com/alecthomas/gometalinter
+- go get -u github.com/go-swagger/go-swagger/cmd/swagger
```

Comment:
Is gometalinter and go-swagger needed? I don't think they are required by `make test`.

Reply:
I wanted to add linting as well and make it already "ready" for the future. I can remove it now. 

I don't know if go-swagger is not needed, easier to remove it and see.

Reply:
Can you remove `go-swagger` so we can see? :) Any dependencies should be included with `glide install`.

Let's keep gometalinter!